daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/TestMain.scala

```diff
@@ -49,36 +75,43 @@ object TestMain extends StrictLogging {
     implicit val materializer: Materializer = Materializer(system)
     implicit val executionContext: ExecutionContext = system.dispatcher
 
-    val (participantParams, participantCleanup) = config.participantConfig match {
```

Comment:
Do I read properly that `participantCleanup` was never doing anything ?

Reply:
Ah right yeah, I had a comment for it previously, but forgot to commit that - yes, it was always set to no-op

Reply:
:+1: